melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/ArbeidsgiverDeserializer.java

```diff
@@ -0,0 +1,31 @@
+package no.nav.melosys.eessi.models.sed;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.nav.melosys.eessi.models.sed.nav.Arbeidsgiver;
+
+public class ArbeidsgiverDeserializer extends JsonDeserializer<List<Arbeidsgiver>> {
```

Comment:
> NOTE: it is recommended that custom serializers extend StdSerializer instead of this class, since it will implement many of optional methods of this class.

Reply:
🤦 :+1: